extension/api.js

```diff
@@ -43,25 +55,51 @@
 	})();
 
 	window.gitHubNotifCount = function (callback) {
-		var tmp = document.createElement('div');
+		var token = GitHubNotify.settings.get('oauthToken');
+		var opts = {
+				Authorization: 'token ' + token
+		};
+		var participating = GitHubNotify.settings.get('useParticipatingCount')
+			? '?participating=true'
+			: '';
+		var url = GitHubNotify.settings.get('rootUrl');
 
-		xhr('GET', GitHubNotify.settings.get('notificationUrl'), function (data, status) {
+		if (!token) {
+			callback(-4);
+			return;
+		}
+		if (/github.com\/?/.test(url)) {
```

Comment:
This should be `/\.github\.com\/?/.test(url)` otherwise the current `.` act as regex wildcard characters.